packages/components/src/internal/renderers/StorageStatusRenderer.tsx

```diff
@@ -14,7 +14,11 @@ export class StorageStatusRenderer extends React.PureComponent<StorageStatusProp
         if (value?.toLowerCase() === 'not in storage' || value?.toLowerCase() === 'discarded') {
             return value;
         } else {
-            return <a href={data.get('url')}>{value}</a>;
+            const href = data?.get('url');
+            if (href)
+                return <a href={data.get('url')}>{value}</a>;
+
+            return value;
```

Comment:
@XingY Ian and I were discussing this over chat. It would be good to try and get the repro to happen on your side, but in looking at this code change if the `data` variable above is undefined then the code falls into this else block in which case the `href` variable will also be defined so the result is `return undefined` which React won't like. Should we instead move the new check to the top of the render function and say `if (!data) return null`?